src/Modules/StatCan.OrchardCore.EmailTemplates/Views/EmailTemplate/Create.cshtml

```diff
@@ -101,3 +101,21 @@
         }
     </div>
 </form>
+
+<script at="Foot">
+    $(function () {
+        var codeMirrorOptions = {
+            autoRefresh: true,
+            lineNumbers: true,
+            styleActiveLine: true,
+            matchBrackets: true,
+            mode: { name: "liquid" },
+        };
+        CodeMirror.fromTextArea(document.getElementById('@Html.IdFor(x => x.AuthorExpression)'), codeMirrorOptions);
```

Comment:
Do you need to change the inputs to TextArea? or does it work with inputs ?

Reply:
Yes this is the only way I found to make it work. It looks like this:
![image](https://user-images.githubusercontent.com/66786602/103015337-3cfcef00-450e-11eb-8b27-56314735c969.png)